docs/topics/proc_deploying-the-booster-using-the-launcher.adoc

```diff
@@ -8,15 +8,18 @@
 // Rationale: This procedure is identical in all deployments
 
 [#deploying-the-booster-using-launcher_{context}]
-= Deploying the Booster Using {launcher}
+ifdef::parameter-openshiftlocal[]
```

Comment:
I believe now is the right time to break this module into two, as right now, the overlap between OSL and OSO content is zero. If we need this merged in quickly, let's file a follow-up issue, but in the long term, I'd like to get rid of such situations.